src/plugins/LADTable/components/LADRow.vue

```diff
@@ -110,24 +112,34 @@ export default {
     },
     methods: {
         updateValues(datum) {
-            let newTimestamp = this.formats[this.timestampKey].parse(datum),
-                limit;
-
-            if(this.shouldUpdate(newTimestamp)) {
-                this.timestamp = this.formats[this.timestampKey].parse(datum);
-                this.value = this.formats[this.valueKey].format(datum);
-                limit = this.limitEvaluator.evaluate(datum, this.valueMetadata);
-                if (limit) {
-                    this.valueClass = limit.cssClass;
-                } else {
-                    this.valueClass = '';
+            if(datum) {
+                let newTimestamp = this.getParsedTimestamp(datum),
+                    limit;
+
+                if(this.shouldUpdate(newTimestamp)) {
+                    this.timestamp = this.getParsedTimestamp(datum);
```

Comment:
I think we can just use `newTimestamp` here, instead of calling the function again? `updateValues` will be called with very high frequency, we need to be very wary of performance here.